### DISCUSSION TITLE
What is transform pseudo-class variant configuration for?

### DISCUSSION THREAD

**Discussion by muhammadsammy** [tailwindlabs/tailwindcss#1788] [Help] [answered]
In TailwindCSS [pseudo-class variants documentation](https://tailwindcss.com/docs/pseudo-class-variants/#default-variants-reference), there is a property named `transform` and then all transforms variants are listed.
What classes does the `transform` property configure?
Thanks!

**Comment by Apostolique** [comment, marked as answer]
I can't answer perfectly, but you can see the code here: https://github.com/tailwindcss/tailwindcss/blob/master/src/plugins/transform.js

Looks like it's used as a base for the other transforms `scale`, `rotate`, `translate`, `skew`, `origin`.

For example with scale:

```html
<img class="transform scale-75 ...">
```

**Reply by estevanmaito** [reply]
That's it.

`transform` just puts in place some rules with CSS custom properties. Then when other transformations are applied, like with `scale`, `rotate`, etc, the default custom properties are overwritten.

The exact properties transformed can be found under the **Transform** category in docs.

**Reply by muhammadsammy** [reply]
Thank you!
